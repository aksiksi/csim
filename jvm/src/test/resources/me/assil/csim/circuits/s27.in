1110101
0001010
1010101
0110111
1010001
