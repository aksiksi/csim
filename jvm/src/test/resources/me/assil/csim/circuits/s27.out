1100
0010
1100
1000
1100